README.md

```diff
@@ -135,22 +135,20 @@ You can read about the formatting guide [here](https://github.com/space-concordi
 For a quick primer on our workflow using git, [CLICK HERE :)](https://github.com/space-concordia-robotics/robotics-prototype/wiki/Git-Workflow-and-Conventions)
 ### Commit message hook
 
-This explains how to use and setup a git hook that will prepend the issue number to a commit message.
-Git hooks are used to automate git commands and functions.
-There are a number of different options, but this particular hook will add the issue number to your
-`git commit -m` message once it is pushed to the repository.
-Git hooks live in a local `.git/hooks` file of each repo, so initializing this hook in the this repo will not change any other repos you might have.
+This explains how to setup git hooks which prepend an issue number to a commit message, and verify branch naming conventions. Git hooks are located in the local `.git/hooks` file of each repo, so initializing this hook in the this repo will not change any other repos you might have.
 
-### Setting up a git hook
+### Setting up git hooks
 
-If you're in Linux run the following command. If you're on windows, install [Git Bash](https://git-scm.com/downloads) to be able to run the same command.
+Git hooks are important for performing repository validity checks. To setup git hooks using Git Bash, run the following commands from the root of the repository (ex: from ~/Programming/robotics-prototype/):
 
-From the root of the repository:
-- `cp commit-message-hook.sh .git/hooks/prepare-commit-msg`
+- `cp commit-message-hook.sh .git/hooks/prepare-commit-msg.sh`
```

Comment:
The destination files should not be ending with `.sh`, only the source files should.
The example below proves it.

```(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$ git status
On branch commit-verification-162
Your branch is up-to-date with 'origin/commit-verification-162'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   branch-name-verification.sh

no changes added to commit (use "git add" and/or "git commit -a")
(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$ git diff
diff --git a/branch-name-verification.sh b/branch-name-verification.sh
old mode 100644
new mode 100755
(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$ git add branch-name-verification.sh 
(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$ git commit -m "ads"
[commit-verification-162 a847767] [#162] ads
 1 file changed, 0 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 branch-name-verification.sh
(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$ echo 1 > test.txt
(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$ git add test.txt 
(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$ cp branch-name-verification.sh .git/hooks/prepare-commit-msg.sh
(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$ rm .git/hooks/prepare-commit-msg
(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$ git commit -m "asdf"
[commit-verification-162 e2a392e] asdf
 1 file changed, 1 insertion(+)
 create mode 100644 test.txt
(venv) beep@beep-Inspiron-15-7000-Gaming:~/Programming/robotics-prototype$
```